desktop-exporter/server.go

```diff
@@ -79,25 +77,26 @@ func traceIDHandler(store *TraceStore) func(http.ResponseWriter, *http.Request)
 }
 
 func indexHandler(writer http.ResponseWriter, request *http.Request){
-	indexBytes, err := assets.ReadFile("static/index.html")
-	if err != nil {
-		writer.WriteHeader(http.StatusInternalServerError)
-            return
-	}
-	writer.Write(indexBytes)
+	http.ServeFile(writer, request, "./desktop-exporter/static/index.html")
+	// indexBytes, err := assets.ReadFile("static/index.html")
+	// if err != nil {
+	// 	writer.WriteHeader(http.StatusInternalServerError)
+    //         return
+	// }
+	// writer.Write(indexBytes)
```

Comment:
You might do something like:

```go
if os.Getenv("SERVE_FROM_FS") == "true" {
    http.ServeFile(writer, request, "./desktop-exporter/static/index.html")
} else {
    indexBytes, err := assets.ReadFile("static/index.html")
    if err != nil {
        writer.WriteHeader(http.StatusInternalServerError)
        return
    }
    writer.Write(indexBytes)
}
```

so you can run during development like:

```
$ SERVE_FROM_FS=true go run ./...
```

Reply:
Thank you!